openedx_external_enrollments/management/commands/migrate_other_course_settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Script for exporting all courses from Mongo to a othercoursesettings table on edxapp database.
+"""
+from django.core.management.base import BaseCommand
+
+from openedx_external_enrollments.edxapp_wrapper.get_course_settings import migrate_course_settings
+
+
+class Command(BaseCommand):
+    """
+    Migrate all courses via django command.
+    """
+    help = 'Export all courses from mongo to migrate into OtherCourseSettings table.'
+
+    def add_arguments(self, parser):
+        """
+        Required command arguments:
+        delay_seconds -> sleep time in seconds.
+        group_length -> number of courses to be migrated by groups.
+        """
+        parser.add_argument('delay_seconds', type=int)
+        parser.add_argument('group_length', type=int)
```

Comment:
Can you add default values to these parameters?| @ivanvgh

Reply:
Sure.

Reply:
Since they will have default values they will change to optional arguments. Done.